### ISSUE TITLE
Continued training `model.fit()` creates spurious validation entries in external log

### ISSUE THREAD

**Issue by fdraxler** [LarsKue/lightning-trainable#10]
In https://github.com/LarsKue/lightning-trainable/blob/main/src/lightning_trainable/trainable/trainable.py#L352, the vall to `validate()` creates an entry in the tensorboard log at step=0, even if the training is continued from a later step. (Happens when `fit_kwargs["ckpt_path"]` is set to a checkpoint path).

**Comment by LarsKue** [comment]
Nicely spotted. I will have a look into if we can adjust the state before the validation call.

**Comment by LarsKue** [comment]
Added a work-around in eb29b1603999c91fa0d4d157a1a139302143901e.

For future reference: this bug was caused by a preemptive call to `trainer.validate()` before fitting. However, we needed to perform this call in order to gather initial metrics to watch for hyperparameter logging (as required by the `TensorBoardLogger`). This is now done via a test batch with no log calls.

**Comment by fdraxler** [comment]
on_train_start misses its target: It does not have the "training/" or "validation/" prefix in the log entries, so that the Tensorboard is cluttered with additional panels that only get a single entry ever.

**Comment by fdraxler** [comment]
I added a preliminary fix for this, but now there are "nan"s in the Tensorboard...

**Comment by LarsKue** [comment]
Thank you for the preliminary fix.

Are there any cases in which we care about the metrics evaluated on the training set? HParams are in my experience only ever optimized on the validation set, so fetching a validation batch and computing the starting keys & values for tensorboard seems like a good solution to me here (of course, now with the proper "validation/" prefix).
